ci/jenkins/templates/conformance-template.yaml

```diff
@@ -13,7 +12,7 @@
     parameters:
         - string:
             name: BRANCH
-            default: '{branch}'
+            default: master
```

Comment:
not sure if this must have quotes: `"master`"

Reply:
mmmh, good point, lets see what jjb prints

Reply:
no need! jjb output looks good.